### ISSUE TITLE
gdc 32bit

### ISSUE THREAD

**Issue by John-Colvin** [travis-ci/travis-ci#4685]
The gdc installation doesn't seem to include 32 bit phobos:
```
/usr/bin/ld: skipping incompatible /home/travis/gdc/bin/../lib/gcc/x86_64-unknown-linux-gnu/4.9.2/../../../libgphobos2.a when searching for -lgphobos2
/usr/bin/ld: cannot find -lgphobos2
```

ping @MartinNowak @klickverbot @ibuclaw

**Comment by jpf91** [comment]
That's correct. It's a standard x86_64-linux-gnu binary build from http://gdcproject.org/downloads and these don't support multilib yet.

We could install the `x86_64-linux-gnu/i686-linux-gnu` compiler as well, but it's a real cross compiler and won't link to libraries installed on the host. (E.g. you can't do -lcurl to link to your system 32bit libcurl).

I'll have a look if we can build multilib compilers for x86.

**Comment by MartinNowak** [comment]
That means bigger downloads? Is it really import for you? Can you maybe download a 32-bit version of gdc yourself?
